Updated header and footer #2

Rushhaabhhh opened the pull request:
![Screenshot 2024-10-01 174201](https://github.com/user-attachments/assets/56034c63-31d1-4236-a220-ee2661140bca)


Please review the UI changes and let me know if any further adjustments are needed.

Rushhaabhhh committed:
Updated header and footer

StephCurry07 commented:
Tbh, the footer is not looking nice....also do you think it's looking good in black?
The logo's etherwheels not displayed due to both dark...ether is darkblue-purple and header is black.
Could you either change the logo or make the header a bit different.
And remove the footer coz it's also engaging with the open assistant button...and also not looking good.

Rushhaabhhh committed:
Updated UI

Rushhaabhhh commented:
![image](https://github.com/user-attachments/assets/4be2e0ca-3aa1-4f90-b3b7-6bd3b984b53e)
@StephCurry07 I have updated the header and removed the footer, please review the changes.

StephCurry07 commented:
umm... previous one looks better... can you change the logo...
also can you combine the change role and current role. Can be adjusted in single nav item

Rushhaabhhh committed:
Header UI changes

Rushhaabhhh commented:
![image](https://github.com/user-attachments/assets/ca49ded1-9ce3-4c32-a1de-d8d81355d8d5)

@StephCurry07  As you asked I have updated the header,  please review the changes.

StephCurry07 commented:
Shift it to right side...where the change role originally was and do like
Change role(Passenger) or changerole(driver) whatever the current role is...the button would work as it was doing earlier. and youre good to go

Rushhaabhhh committed:
Update Header

Rushhaabhhh commented:
![image](https://github.com/user-attachments/assets/6a6b586e-c9e7-498d-83c8-557528d66db4)

Changes made

Rushhaabhhh committed:
Merge branch 'main' into main

Rushhaabhhh commented:
@StephCurry07 Any updates on the PR ?

StephCurry07 commented:
Yea I need to test it....Also look at the font pls. Make sure it's uniform.

StephCurry07 commented:
Change the font size... and remove the change role dropdown on click. Once the user clicks he gets redirected.
